vignettes/GenomicFiles.Rmd

```diff
@@ -0,0 +1,687 @@
+---
+title: "Introduction to GenomicFiles"
+authors:
+  - name: "Valerie Obenchain"
+  - name: "Michael Love"
+  - name: "Martin Morgan"
+  - name: "Sonali Kumari"
+    affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date: "Last modified: October 2014; Compiled: `r format(Sys.time(), '%B %d , %Y')`"
+
+vignette: >
+  %\VignetteIndexEntry{Introduction to GenomicFiles}
+  %\VignetteDepends{GenomicAlignments, RNAseqData.HNRNPC.bam.chr14}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+output:
+  BiocStyle::html_document
+---
+
+```{r echo=FALSE, include=FALSE}
+if (!require("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+BiocManager::install("BiocStyle", force = TRUE)
+```
```

Comment:
Remove 20-24. If you read the Bioconductor developer documentation, it states that no package should automatically install packages. This is to give the user full control of their system, which is why all dependencies are documented in the DESCRIPTION file. Remove any extra blank lines. (Generally, we want 1 blank line surround paragraphs or code chunks for readability.)

Reply:
well noted. Thanks for always giving valuable advice.